docker/eidangws_config

```diff
@@ -0,0 +1,161 @@
+# This is <eidangws_config>
+
+[CONFIG_FEDERATOR]
+# Configuration section for the EIDA NG Federator Webservice. Note, that
+# commandline parameters passed is given priority over options set within this
+# file. The syntax for option setting is:
+# 
+# key=value
+#
+# You obtain additional information as well as default configuration values
+# when entering:
+#
+# $ eida-federator -h
+# 
+# See also the Python configparser module documentation:
+#   - https://docs.python.org/library/configparser.html
+#
+# -----------------------------------------------------------------------------
+# Start the application by means of the local Flask WSGI server. 
+# (default: False)
+#
+# start_local={True,False}
+# ----
+# Set the port number the local Flask WSGI server is serving. Note, that this
+# parameter is taken only into consideration if start_local=True.
+# 
+# port=PORT
+# ----
+# Set the routing service url and identifier. (default:
+# http://geofon.gfz-potsdam.de/eidaws/routing/1/)
+#
+# For a complete list of routing services see:
+# https://www.orfeus-eu.org/data/eida/webservices/
+#
```

Comment:
The comments regarding the `routing` configuration parameter are not valid anymore. See also: e45f265a667f18986874e0af8e4837509466fdfe